hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,32 +1,30 @@
-# USB mass storage device boot
+# USB mass storage boot
 
-**USB mass storage boot is available on Raspberry Pi 3B, 3B+, 3A+, and 2B v1.2 only.**
+**USB mass storage boot is available on Raspberry Pi 2B v1.2, 3A, 3A+, 3B, 3B+ only. Support for USB mass storage boot will be added to the Pi 4B in a future software update.**
 
-This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or USB hard disk. Be warned that this feature is experimental and does not work with all USB mass storage devices. See [this blog post](https://www.raspberrypi.org/blog/pi-3-booting-part-i-usb-mass-storage-boot/) from Gordon Hollingworth for an explanation of why some USB mass storage devices don't work, as well as for some background information.
+This tutorial explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or USB hard disk. Note that this feature does not work with all USB mass storage devices.
```

Comment:
Is it worth keeping the link to that blog post?

Reply:
I removed the link to the blog post since it says the USB host boot features are still in beta, and because it says to use the branch=NEXT firmware from rpi-update. We're not recommending rpi-update anymore for anything, and it doesn't need the branch=NEXT firmware either.

The rest of that blog post is actually has really useful discussion about USB host mode boot. Perhaps we should just copy it into a new page in the docs?